pkg/bazel/bazelisk.go

```diff
@@ -256,33 +255,13 @@ func (bazelisk *Bazelisk) getBazelVersion() (string, error) {
 	}
 
 	if len(bazelisk.workspaceRoot) != 0 {
-		bazelVersionPath := filepath.Join(bazelisk.workspaceRoot, ".bazelversion")
-		if _, err := os.Stat(bazelVersionPath); err == nil {
-			f, err := os.Open(bazelVersionPath)
-			if err != nil {
-				return "", fmt.Errorf("could not read %s: %v", bazelVersionPath, err)
-			}
-			defer f.Close()
-
-			scanner := bufio.NewScanner(f)
-			scanner.Split(bufio.ScanLines)
-			scanner.Scan()
-			bazelVersion := scanner.Text()
-			// If the first line of .bazelversion is Aspect CLI, then when we call
-			// bazelisk it will read the file again and we'll call ourselves in a loop.
-			// We don't want to be re-entrant, so detect when our fork is selected.
-			// In this case, the user should put their bazel version on the following line.
-			if strings.HasPrefix(bazelVersion, "aspect-build/") {
```

Comment:
don't we need to keep this feature?

Reply:
Yep. I consolidated it to the `Version` struct and its associated functions. I was DRYing it up, because `bootstrap` needs to read and write the `.bazelversion` file.